docs/source/new-docs/getting-started/system-requirements/index.rst

```diff
@@ -3,30 +3,71 @@
+  GeoIPS.
+- Handling of currently unsupported data sources or formats.
 
-     - How to install GeoIPS
-     - Notes about potential future system requirements, unless absolutely nessesary
-     - Info on how GeoIPS works
+If you plan to use GeoIPS with custom tooling, unsupported datsets, in a
+parallel/batch environment, we recommend that you perform your own testing to
+determine your requirements, especially for production settings.
 
-   - Does include
+User System Requirements
+------------------------
 
-     - "Minimum" GeoIPS system requirements
-     - Recommended GeoIPS system requirements
-     - Here is an example page: `Autodesk Inventor 2025
-       <https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/System-requirements-for-Autodesk-Inventor-2025.html>`_
-     - This is a Reference page - please consult `this guide <https://docs.divio.com/documentation-system/reference/>`_
++------------------+---------------------------------------------------------+
+| Operating System | Minimum Version                                         |
++==================+=========================================================+
+| Linux            | RedHat 8, AlmaLinux 8, RockyLinux 8 equivalent or later |
++------------------+---------------------------------------------------------+
+|                  | Debian 10 equivalent or later                           |
++------------------+---------------------------------------------------------+
+| MacOS            | 12 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
+| Windows          | 10 or later *(not officially supported)*                |
++------------------+---------------------------------------------------------+
 
- - A PR from your feature branch to ``main`` 😊
++----------+-------------+-------------+--------------------------------------+
+| Hardware | Minimum     | Recommended | Supports                             |
++==========+=============+=============+======================================+
+| CPU      | 2 Cores     | 4 Cores     | A single concurrent GeoIPS process   |
++----------+-------------+-------------+--------------------------------------+
+|| Memory  || 12GB       || 128GB      || Production of most GeoIPS imagery.  |
+||         ||            ||            || Some Geostationary imagery requires |
+||         ||            ||            || > 90GB RAM.                         |
++----------+-------------+-------------+--------------------------------------+
+| Storage  | 20GB on SSD | 20GB on SSD | Base GeoIPS system installation.     |
++----------+-------------+-------------+--------------------------------------+
+||         || 20GB       || As needed  || Additional disk space for input and |
+||         ||            ||            || output data storage. May be SDD,    |
+||         ||            ||            || HDD, or NAS.                        |
```

Comment:
@jsolbrig Note we have some system requirements information in the SRS.  Do we want to consolidate this infomation somewhere (maybe the SRS could point to a link if we wanted to keep it separate).  Also, what else do you expect to go in the "getting-started/system-requirements" docs directory?

Reply:
That sounds like a good idea. I pulled a lot of this out of thin air, though I did collect the storage requirements based on my installation.

I think this is the only thing that will go into this directory. You might talk with @biosafetylvl5 about the organization of the documentation if you want to learn more about how they're handling it.

Reply:
What is the SRS @mindyls ?

As for what else is in this dir, not much beyond this `.rst` file. We have an issue up to replace the dirs: https://github.com/NRLMMD-GEOIPS/geoips/issues/603

Reply:
@mindyls I think the SRS needs to be updated in some spots. I think that linking into the documentation for some things might be good to avoid maintaining the same documentation in multiple places. I've updated this section to more closely match what is in the SRS, but this also provides more info than the SRS does. The SRS only talks about number of processors (2 minimum) and RAM (16GB minimum) but doesn't mention storage or recommended resources.

I recommended 128GB of RAM because holding one full ABI dataset in memory at full resolution requires about 90GB of RAM.